### DISCUSSION TITLE
Unable to see LFC appended to the same table after pairwiseDESeq2

### DISCUSSION THREAD

**Discussion by aishwarya-gondane** [erhard-lab/grandR#18] [Q&A]
Hi,

I have recently started using GrandR for the analysis of my SLAM-seq data. I am following the tutorials posted online. I was able to calculate the LFC, when I try to use the PairwiseDESeq2function, it is expected that the results are appended to the existing table from LFC. The LFC results are overwritten by the PairwiseDESeq2 function. 
Thanks in advance!

Best wishes,
Aishwarya

**Comment by florianerhard** [comment]
Hi Aishwarya,
is this with the data set from the vignettes? Could you post the output of `Analyses(data,description=TRUE)` from before and after calling `PairwiseDESeq2()`?
Best,
Florian

**Reply by aishwarya-gondane** [reply]
Hi Florian,

I managed to figure out the issue. I was calling the PairwiseDESeq2 on my data directly, instead of using the LCF variable. Below is the code I was using:

 fold_change_LCF <-  LFC(data, name.prefix = "QC",contrasts=contrasts)
fold_change_LCF <- PairwiseDESeq2(data,name.prefix = "QC",contrasts=contrasts)

Now I managed to figure out the issue and below is my code now 
 fold_change_LCF <-  LFC(data, name.prefix = "QC",contrasts=contrasts)
fold_change_LCF <- PairwiseDESeq2(fold_change_LCF, name.prefix = "QC",contrasts=contrasts)

Thanks!
Best wishes,
Aishwarya
